### ISSUE TITLE
Tracks downloaded to container root instead of MUSIC_STORAGE_BASE_PATH

### ISSUE THREAD

**Issue by lavinir** [kamilkosek/jellyplist#58]
Hi,

I'm on 0.1.9. When adding playlists, the downloads are saved directly to the container root and not the MUSIC_STORAGE_BASE_PATH.
So in my .env file I have:
SPOTDL_OUTPUT_FORMAT = /{artist}/{album}/{title}
MUSIC_STORAGE_BASE_PATH = /mnt/media/Music

The expectation is the files will be saved in : /mnt/media/Music/{artist}/{album}/{title}.
What happens is they instead are just saved in /{artist}/{album}/{title}

I checked inside the container to make sure the env vars exist there and they are indeed defined properly.

Am I doing anything wrong in these definitions?

Thanks for your work on this.

**Comment by kamilkosek** [comment]
can you do the following: 

on your host system, type the following in the shell:
`touch /mnt/media/Music/testfile.test` 

and then inside the container, verify that the file exists:
`ls /mnt/media/Music/testfile.test`

**Comment by lavinir** [comment]
Yes sure, I already saw my existing library there (but did the test anyway). The binding is working, the problem is that the file downloads INSIDE the container are not saved to the correct folder. 
To clarify, I meant they are downloaded inside the root ('/') folder of the container itself instead of the mount directory..

Hope that makes sense.

**Comment by gdshadow01** [comment]
Had the same problem, fixed it with a "/" at the end of MUSIC_STORAGE_BASE_PATH
e.g. MUSIC_STORAGE_BASE_PATH = '/mnt/media/Music/' and no "/" at start of  SPOTDL_OUTPUT_FORMAT, e.g. SPOTDL_OUTPUT_FORMAT = "{artist}/{album}/{title}"

**Comment by lavinir** [comment]
That's crazy, I actually considered trying that but figured it wouldn't make any difference and was following the sample in the documentation. 
Anyway worked, thanks for the tip!

**Comment by kamilkosek** [comment]
Fun fact: I am relatively new to Python and naively thought, that when using `os.path.join()` , the engine would automatically sanitize the folder paths , like in any `.net` language ...

**Comment by lavinir** [comment]
I would have thought the same :)
